### PR TITLE
Completing std/node/fs

### PR DESCRIPTION
**Pull request by disizali**
at the first commit of completing std/node api's , i added exists and existsSync functions to `std/node/fs `module .

It makes me proud to work on this module .

### CODE REVIEW

File: std/examples/chat/server.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ listenAndServe({ port: 8080 }, async (req) => {
       });
     } else {
       // server launched by deno run ./server.ts
-      const file = await Deno.open(u.pathname);
```
Comment (ry) [inline std/examples/chat/server.ts:49]: Unrelated changes

Comment (disizali) [reply std/examples/chat/server.ts]: reverted 👍
